.eslintrc

```diff
@@ -2,6 +2,9 @@
   "env": {
     "node": true
   },
+  "parserOptions": {
+    "ecmaVersion": 6
```

Comment:
did that because my eslint was going nuts :) since we do support only node 6+ - that shouldnt be a problem at all, and I assume once codebase has been completely migrated to ts eslint would go completely